src/lib/seam/access-codes/use-access-codes.ts

```diff
@@ -29,9 +29,7 @@ export function useAccessCodes(
     queryKey: ['access_codes', 'list', normalizedParams],
     queryFn: async () => {
       if (client == null) return []
-      return await client.accessCodes.list(normalizedParams)
-    },
-    onSuccess: (accessCodes) => {
+      const accessCodes = await client.accessCodes.list(normalizedParams)
       for (const accessCode of accessCodes) {
```

Comment:
Why do remove `onSuccess` here but are able to keep using it in the other hooks?

Reply:
`onSuccess` has been removed from queries but not from mutations